.travis.yml

```diff
@@ -0,0 +1,86 @@
+# Travis integration file.
+# Seattle supports Python 2.5 through 2.7, per
+#   seattle.poly.edu/wiki/ProgrammersPage
+#
```

Comment:
ProgrammersPage only mentions Python v2.5/v2.6
Either change on ProgrammersPage or link to https://seattle.poly.edu/wiki/BuildInstructions